translator/internaldata/metrics_to_oc.go

```diff
@@ -413,13 +413,13 @@ func labelValuesToOC(labels pdata.StringMap, labelKeys *labelKeysAndType) []*ocm
 	}
 
 	// Visit all defined labels in the point and override defaults with actual values
-	labels.Range(func(k string, v string) bool {
+	labels.Range(func(k string, v pdata.AttributeValue) bool {
 		// Find the appropriate label value that we need to update
 		keyIndex := labelKeys.keyIndices[k]
 		labelValue := labelValues[keyIndex]
 
 		// Update label value
-		labelValue.Value = v
+		labelValue.Value = v.StringVal() //TODO: should be converted here
```

Comment:
Link TODO to https://github.com/open-telemetry/opentelemetry-collector/issues/3815